pkg/network/node/egressip.go

```diff
@@ -136,7 +136,12 @@ func (eip *egressIPWatcher) ClaimEgressIP(vnid uint32, egressIP, nodeIP string)
 	if nodeIP == eip.localIP {
 		mark := getMarkForVNID(vnid, eip.masqueradeBit)
 		eip.iptablesMark[egressIP] = mark
-		if err := eip.assignEgressIP(egressIP, mark); err != nil {
+		if err := utilwait.PollImmediate(10*time.Millisecond, 2*time.Second, func() (bool, error) {
+			if err := eip.assignEgressIP(egressIP, mark); err != nil {
+				return false, err
```

Comment:
This will not do what you want it to. You need to `return false, nil`. From the doc of `PollImmediate`:

```
PollImmediate tries a condition func until it returns true, an error, or the timeout is reached.
```

Hence if we need to retry we can't return an error when we experience one. 

Moreover, maybe do the same for `releaseEgressIP`? If this is a problem for assign, then it might also be for remove. 

And third: maybe log the error inside `PollImmediate` so that we know what fails every time something is tried, instead of just the final timeout error.

Reply:
done